charts/mothership/values.yaml

```
@@ -59,3 +59,9 @@ blockscout:
   gethRpc: http://node-1:8545
   db:
     storage: 10Gi
+
+bundler:
+  image: stackupwallet/stackup-bundler:v0.6.47
+  loadBalancerExternal: false
+  gethRpc: http://node-1:8545
+  port: 4337
```

+ it'd be good to have default `count: 0` value for ease of migration of existing deployments

---

`count` changed to `enabled`, 
```suggestion
bundler:
  enabled: false
  image: stackupwallet/stackup-bundler:v0.6.47
  loadBalancerExternal: false
  gethRpc: http://node-1:8545
  port: 4337
```